### PR TITLE
Update metricflow-cli.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding how to add dimension filter to where filter per [slack thread](https://getdbt.slack.com/archives/C02CCBBBR1D/p1692232617577309)

### CODE REVIEW

File: website/docs/docs/build/metricflow-cli.md [hunk 1]
```diff
@@ -140,44 +140,60 @@ Create a new query with MetricFlow, execute that query against the user's data p
...
+
   --end-time TEXT          Optional iso8601 timestamp to constraint the end
                            time of the data (inclusive)
+
   --start-time TEXT        Optional iso8601 timestamp to constraint the start
                            time of the data (inclusive)
+
   --where TEXT             SQL-like where statement provided as a string. For
                            example: --where "revenue > 100"
+                           To add a dimension filter to a where filter, you have to indicate that the filter item is part of your model.                            Reference the dimension using the following template wrapper: {{ Dimension(‘model_name1__model_name2’) }}
```
Comment (Jstein77) [inline website/docs/docs/build/metricflow-cli.md]:
```suggestion
                           To add a dimension filter to a where filter, you have to indicate that the filter item is part of your model.                            Reference the dimension using the following template wrapper: {{ Dimension(‘primary_entity__dimension_name’) }}
```

**Comment by Jstein77** [comment]
@mirnawong1 thanks for opening this PR! Can we also update the where clause example in the query example section? 
The example should be: 
```
mf query --metrics order_total --group-by metric_time --where "{{Dimension('order_id__is_food_order')}} = True"
```

I think we should also update the metric in the other examples to `order_total` so it's consistent with the example project.

**Comment by mirnawong1** [comment]
hey @Jstein77 ,t hanks so much! can you give this another look?

**Review by Jstein77 — APPROVED** [review]
LGTM!

**Pull request merged**